src/pages/Pokemon/Evolution.js

```diff
@@ -1,10 +1,35 @@
 import React from "react";
-import Image from "./Image";
+import { useNavigate } from "react-router-dom";
+import { formatName, formatNumber } from "../../utils";
+import Type from "../../components/Type/Type";
+import "./pokemon.css";
+
+const Evolution = ({ pokemonItem }) => {
+  let navigate = useNavigate();
+  const evolutions = pokemonItem?.evoListItems?.map((item) => (
```

Comment:
doing it this way is fine mate but I'm more a fan of making things that return HTML elements as actual components rather than just a const value like here. Reason being is it's easier to isolate components and also just looks nicer in my opinion. But yeah tbh you can do it either way so YAAAAAAAAAAAAAAAAAAAAAAAH